src/configure/templates/githubWorkflowTemplates/nodejsWithGruntOnLinuxWebApp.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     # install dependencies, build, and test
     - name: npm install, build
-      working-directory: {{{ repositoryAnalysisApplicationSettings.settings.nodePackageFileDirectory }}}
+      working-directory: {{{ workingDirectory }}}
       run: |
         npm install
         grunt --gruntfile gruntfile.js
```

Comment:
this should be updated to point to applicationSettings setting

Reply:
similarly for others as well

Reply:
No, working directory is not a part of applicationSettings since we are setting it as a property of pipeline configuration.